pytorch/inference/buildspec-1-12.yml

```diff
@@ -34,10 +34,12 @@ images:
     image_size_baseline: 4899
     device_type: &DEVICE_TYPE cpu
     python_version: &DOCKER_PYTHON_VERSION py3
+    jdk_version: &DOCKER_JDK_VERSION jdk17
     tag_python_version: &TAG_PYTHON_VERSION py38
+    tag_jdk_version: &TAG_JDK_VERSION jdk17
     os_version: &OS_VERSION ubuntu20.04
-    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *TAG_JDK_VERSION, "-", *OS_VERSION, "-ec2" ]
```

Comment:
Please remove jdk version from tag - this will have an impact on release process and isn't something we can easily introduce